Add initial version of OOPS team etiquette

Cruikshanks opened the pull request:
New members to the team will benefit from knowing what the existing team considers to be good office etiquette. This should reduce the chance of conflict or resentment.

This change adds an initial version of a guide on etiquette.

Cruikshanks committed:
Add initial version of OOPS team etiquette

New members to the team will benefit from knowing what the existing team considers to be good office etiquette. This should reduce the chance of conflict or resentment.

This change adds an initial version of a guide on etiquette.

Cruikshanks committed:
Update root README to include link to etiquette

Texel22 commented on guides/etiquette.md:
```diff
@@ -0,0 +1,17 @@
+
+To ensure homgenous team working and enthusiastically transform real-time partnerships we have produced this guide on team ettiquette.
+
+## Avoid idle gossip
```
Sometimes gossip is good for morale.

Cruikshanks replied:
Completely agree 👍

Cruikshanks replied:
I should add this to the guide!

Cruikshanks committed:
Update Etiquette guide based on feedback

Good suggestion from @Qbtron22 regards morale of gossip so have added to the guide.
